database/migrations/2019_02_03_133007_change_id_type.php

```diff
@@ -40,16 +40,25 @@ public function up()
 		]);
 
 	    Schema::table('user_album', function (Blueprint $table) {
-		    $table->dropForeign(['album_id']);
-		    $table->dropColumn('album_id');
+        if (DB::getDriverName() !== 'sqlite') {
+          $table->dropForeign(['album_id']);
+        }
+        $table->dropForeign(['album_id']);
```

Comment:
you need to remove that line. You have a duplicate query here. It will try to drop twice which will break (most likely the reason why travis is complaining).

Reply:
reverted